### DISCUSSION TITLE
Bitwarden Client 2023.7.0 not working (New Client Update)

### DISCUSSION THREAD

**Discussion by zandercodes** [dani-garcia/vaultwarden#3692] [Q&A] [answered]
Bitwarden has released a new client 2023.7.0.
With it everything is displayed empty in the vault.
The version 2023.5.0 still works everything.

As a solution I suggest temporarily to install the version 2023.5.0 until the new version of bitwarden is supported.

**Comment by BlackDex** [comment]
Which client?

**Comment by zandercodes** [comment]
Chrome Browser Extension and Windows Client 2023.7.0

**Comment by Gurkengewuerz** [comment]
Similar here. after restarting my browser the client was logged out. a login was possible but changing anything in the vault leads to a infinite loading animation. something strange with the version 2023.7.0.
I am using the Edge Browser Extension.

**Comment by BlackDex** [comment]
I just tested it, and it works fine for me, Both desktop and chrome extension.

**Comment by BlackDex** [comment]
There is nothing changed into the workings as far as i can tell. I even tested the web-vault 2023.7.0 (As did others) and nothing has changed there either. Since all the clients (except mobile) use the exact same base I think it is strange that it doesn't work for you.

**Comment by vherrlein** [comment]
From my side, the extension in Edge is slow with a spin icon each time you want to open it, and autofill doesn't work anymore
![image](https://github.com/dani-garcia/vaultwarden/assets/24614045/de884307-ae35-43b1-a634-05679b808e3f)
Version:;
![image](https://github.com/dani-garcia/vaultwarden/assets/24614045/5cbdb3e6-e1cc-45d0-ade7-7bc3fe5bbde6)

Fiddler show nothing, websocket opened, not unusual logs in vaultwarden

**Comment by BlackDex** [comment]
Any logs on the Vaultwarden side or reverse proxy which might indicate something?

**Comment by BlackDex** [comment]
Could you also try `vault.bitwarden.com` if it happens there also? I just can't replicate it

**Comment by Bapf** [comment]
Having the same issue.

**Comment by BlackDex** [comment, marked as answer]
See: https://github.com/bitwarden/clients/issues/5807

**Comment by leranp** [comment]
is there any problem with the Android app ? after the update the app is crashing when login

**Reply by zandercodes** [reply]
Delete the Cache and Appdata and relogin.
